include/aspect/stokes_matrix_free.h

```diff
@@ -530,6 +530,8 @@ namespace aspect
 
       Simulator<dim> &sim;
 
+      bool options_print_details;
```

Comment:
This needs a comment. What does the `options_` prefix indicate?

Reply:
Not sure, that it is an option? :-) I removed it.